files/management/commands/populate_topics.py

```
@@ -0,0 +1,71 @@
+            action="store_true",
+            help="Show what would be created without making changes",
+        )
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        self.stdout.write(self.style.NOTICE("Starting Topic population..."))
+
+        created_count = 0
+        updated_count = 0
+
+        for topic_code, topic_title in lists.video_topics:
```

Ruff (0.13.3)

Loop control variable `topic_code` not used within loop body

Rename unused `topic_code` to `_topic_code`

(B007)